developer/debugging/test-bench.md

```diff
@@ -30,53 +30,55 @@ Internet access is intentionally disabled by default in dom0. But to ease the de
 
 1. Remove the network card (PCI device) from `sys-net`
 2. Restart your computer (for the removal to take effect)
-3. The following script should enable your network card in dom0. *Be sure to adjust the script's variables to suit your needs.* You'll need to run this at every startup (TODO: describe how to run this at every startup).
-
-    ```bash
-    #!/bin/sh
-
-    # adjust this for your NIC (run lspci)
-    BDF=0000:02:00.0
-
-    # adjust this for your network driver
-    DRIVER=e1000e
-
-    prog=$(basename $0)
-    
-    pciunbind() {
-        local path
-        path=/sys/bus/pci/devices/${1}/driver/unbind
-        if ! [ -w ${path} ]; then
-            echo "${prog}: Device ${1} not bound"
-            return 1
-        fi
-        echo -n ${1} >${path}
-    }
-
-    pcibind() {
-        local path
-        path=/sys/bus/pci/drivers/${2}/bind
-        if ! [ -w ${path} ]; then
-            echo "${prog}: Driver ${2} not found"
-            return 1
-        fi
-        echo ${1} >${path}
-    }
-
-    pciunbind ${BDF}
-    pcibind ${BDF} ${DRIVER}
+3. Install `dhcp-client` and `openssh-server` on your testbench's dom0.
+4. The following script should enable your network card in dom0. *Be sure to adjust the script's variables to suit your needs.* You'll need to run this at every startup with sudo (TODO: describe how to run this at every startup).
```

Comment:
I do this via `/etc/systemd/system/dom0-network-direct.service`:
```
[Unit]
Description=Connect network to dom0

[Service]
Type=oneshot
ExecStart=/home/user/bin/dom0_network.sh

[Install]
WantedBy=multi-user.target
```
(adjust script path as necessary)
And then call `systemctl enable dom0-network-direct`.

Reply:
Perfect. Just added this.